pkgs/applications/video/mpv/scripts/videoclip.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, stdenvNoCC
+, curl
+, xclip
+, wl-clipboard
+, stdenv
+,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "videoclip";
+  version = "unstable-2023-11-01";
+
+  src = fetchFromGitHub {
+    owner = "Ajatt-Tools";
+    repo = "videoclip";
+    rev = "509b6b624592dbac6f2d7e0c4bc92a76460e7129";
+    hash = "sha256-fMuoBWM0bmRhdRoGxLYsqBjy2H+AVwOvVeva3SBR/EA=";
+  };
+
+  dontBuild = true;
+
+  patchPhase =
+    ''
+      substituteInPlace platform.lua \
+      --replace \'curl\' \'${lib.getExe curl}\' \
+    ''
+    + lib.optionalString stdenv.isLinux ''
+      --replace xclip ${lib.getExe xclip} \
+      --replace wl-copy ${lib.getBin wl-clipboard}/bin/wl-copy
+    '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/mpv/scripts/videoclip
+    cp *.lua $out/share/mpv/scripts/videoclip
+
+    runHook postInstall
+  '';
+
+  passthru.scriptName = "videoclip";
+
+  meta = with lib; {
+    description = "Easily create videoclips with mpv.";
```

```suggestion
    description = "Easily create videoclips with mpv";
```

never end desc with a period as per nixpkgs manual